src/sdl2/ttf/font.rs

```diff
@@ -308,7 +308,7 @@ impl<'ttf, 'r> Font<'ttf, 'r> {
     // this can prevent introducing UB until
     // https://github.com/rust-lang/rust-clippy/issues/5953 is fixed
     #[allow(clippy::trivially_copy_pass_by_ref)]
-    unsafe fn raw(&self) -> *mut ttf::TTF_Font {
+    pub unsafe fn raw(&self) -> *mut ttf::TTF_Font {
```

Comment:
Just obtaining raw pointers is not unsafe, `unsafe` keyword can be removed.

Reply:
thanks 1969b78beaa2c27413f2900ece4f26bbe0e4f801